src/wasm/ExportedFunctionObject.cpp

```diff
@@ -120,7 +120,7 @@ static Value callExportedFunction(ExecutionState& state, Value thisValue, size_t
     // Otherwise,
     // Let values be << >>.
     ValueVector values;
-    values.resizeWithUninitializedValues(ret.size);
+    values.resizeFitWithUninitializedValues(ret.size);
```

Comment:
What about replacing all the use cases of `resizeWithUninitializedValues` by `resizeFitWithUninitializedValues`?
IMO its better to replace them all because the most case of `resizeWithUninitializedValues` just use the initial size with no extension. Correct this if wrong.

Reply:
Some cases use TightVector. so there is no resizeFitWithUninitializedValues needs :(